homework3/task3.js

```diff
@@ -0,0 +1,27 @@
+const splittedString = str => {
+    
+    let arr = str.split(",")
+    let arrWithStr = [];
+    let arrWithNumber=[];
+    
+    arr.forEach(function(arr){
+        let elNum = Number(arr);
+        
+        if (isNaN(elNum)) {
+            arrWithStr.push(arr);
```

Comment:
'Array with words:': [ 'Red', '  Green', '  Blue', ' White' ] - think how to remove spaces before 'Green' and  'Blue'
Expected result: [ 'Red', ' Green', ' Blue', ' White' ]